public/directives/wz-table/wz-table-directive.js

```diff
@@ -487,23 +492,48 @@ app.directive('wzTable', function() {
       };
 
       $scope.showTooltip = (id1, id2, item) => {
-        const $element = $('#td-' + id1 + '-' + id2 + ' div');
-        const $c = $element
-          .clone()
-          .css({ display: 'inline', width: 'auto', visibility: 'hidden' })
-          .appendTo('body');
-        if (
-          $c.width() > $element.width() &&
-          (($element || [])[0].children || [])[0].innerText !== '-'
-        ) {
+        const $element = $('#td-' + id1 + '-' + id2 + ' div span.wz-text-truncatable');
+        if($element[0].offsetWidth < $element[0].scrollWidth){
           if (!item.showTooltip) {
             item.showTooltip = [];
           }
           item.showTooltip[id2] = true;
         }
-        $c.remove();
       };
 
+      const filterableColumns = () => {
+        $scope.filterableColumns = [];
+        $scope.keys.forEach(k => {
+          const key = $scope.parseKey(k);
+          const canFilter = (($scope.path === '/rules' && (key === 'level' || key === 'pci' || key === 'gdpr' || key === 'groups' || key === 'file' || key === 'path')) ||
+            ($scope.path === '/decoders' && (key === 'path' || key === 'file')))
+          if (canFilter) {
+            $scope.filterableColumns[key] = true;
```

Comment:
Remove the `if`, and use the above variables:
`$scope.filterableColumns[key] = !!(var1 || var2 || var3..)`